### PR TITLE
Edit/follow up required and call count display changes

### PR DESCRIPTION
**Pull request by Duslerke**
# What:
- Added "Follow-up required?" column to the "Calls list" page.
- Removed "Total call status" information block from the edit view.
- Removed "Calls with resident" field from the remaining calls information block.
- Replaced "Follow up requested date" with "Follow up required: Y" in the remaining calls information block.
- Renamed "Initial requested date" field into "Latest call request date" in the remaining calls information block.
- Centered "Unsuccessful call attempts" table head as well as the values.
- Centered "Follow-up required" table head as well as the values.

# Why:
- "Total call status" was considered Non-essential.
- "Follow up requested date' is not needed as a follow up doesn't necessarily refer to a call.
- Reasons behind the other changes were not specified, but might be linked to decluttering and making easier to read the interface.

# Screenshots:
**Callbacks list:**
image
**Help request edit view:**
image

# Notes: 
- Trello card link: https://trello.com/c/hSfaTcVA/156-call-outcomes-view-changes

### CODE REVIEW

File: views/partials/help-request-data-list.njk [hunk 1]
```diff
@@ -35,7 +36,9 @@
                 {% endfor %}
                 </span>
                 {% set redText = "redText" if unsuccessfulCalls>4 %}
-                <td class="govuk-table__cell {{redText}}">{{unsuccessfulCalls}}</td>
+                <td style="text-align:center" class="govuk-table__cell {{redText}}">{{unsuccessfulCalls}}</td>
+
+                <td style="text-align:center" class="govuk-table__cell">{% if item.CallbackRequired %}Y{% endif %}</td>
```
Comment (kosiakkatrina) [inline views/partials/help-request-data-list.njk:41]: Can we extract this into a css file?

File: views/help-requests/help-request-edit.njk [hunk 2]
```diff
@@ -92,60 +92,35 @@
...
-            </div>
-        </div>
          <div class="govuk-grid-row">
             <div class="govuk-grid-column-one-half">
                 <p class="lbh-body-m">
                     <strong>Current call request status</strong><br />
-                    Initial requested date: {{query.creation_date}} <br />
-                    {%if followup %}
-                        Follow up requested date: {{followupRequestedDates|first}} <br />
+                    Latest call request date: {{query.creation_date}} <br />
```
Comment (kosiakkatrina) [inline views/help-requests/help-request-edit.njk:120]: Hm, not sure if it's just me but wording it as latest call request date seems like it's supposed to be mutable 😅

Comment (Duslerke) [reply views/help-requests/help-request-edit.njk:120]: I had this question as well, however it seems like it has to be a creation date 🤔 
https://docs.google.com/presentation/d/1esSL_HeJqiBmhI_ir4LmgCoPHDEdignzFLIivzSFDO8/edit?disco=AAAAKsjpa_4

File: views/partials/help-request-data-list.njk (hunk 1 above)
Comment (Duslerke) [reply views/partials/help-request-data-list.njk:41]: Maybe. I see… there is this `app/assests/scss/app.scss` file - guess I could drop line into a class .cell-centered-text there. If it gets loaded after the govuk styles, then this rule might just override the govuk-table__cell's default text-align:left

**Review by kosiakkatrina — APPROVED** [review]

File: views/partials/help-request-data-list.njk (hunk 1 above)
Comment (Duslerke) [reply views/partials/help-request-data-list.njk]: Done!

**Review by kosiakkatrina — APPROVED** [review]

**Pull request merged**